opentelemetry-appender-tracing/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+- [1869](https://github.com/open-telemetry/opentelemetry-rust/pull/1869) Utilize the `LogRecord::set_target()` method to pass the tracing target to the SDK.
```

Comment:
add a note saying that, exporters might use target to override the instrumentation scope, which previously contained "opentelemetry-appender-tracing".